src/editors/document/org/OrgDetailsEditor.tsx

```diff
@@ -85,6 +47,28 @@ const enum TABS {
   Actions = 3,
 }
 
+/*
```

Comment:
We generally do not leave commented out code in place.  Since we can always access code from revision history, just remove the commented out code.

Reply:
OK, done.